src/app/components/card/card.component.ts

```diff
@@ -83,25 +105,10 @@ export class CardComponent {
         return this.card.types.map<string>((type: CardType) => type.name).join(' - ');
     }
 
-    get color(): string {
-        const orderedPrimaryCardTypes = [
-            CardTypeId.Treasure,
-            CardTypeId.Duration,
-            CardTypeId.Reserve,
-            CardTypeId.Victory,
-            CardTypeId.Reaction,
-            CardTypeId.Shelter,
-            CardTypeId.Event,
-            CardTypeId.Ruins,
-            CardTypeId.Landmark,
-            CardTypeId.Project,
-            CardTypeId.Way,
-            CardTypeId.Trait,
-            CardTypeId.Ally,
-        ];
+    get background(): string {
         const orderedSecondaryCardTypes = [CardTypeId.Action, CardTypeId.Night];
```

Comment:
make it a field like `orderedPrimaryCardTypes` for more consistency

Reply:
done